evm/README.md

```diff
@@ -76,6 +76,27 @@ ts-node evm/deploy-its -e testnet -n ethereum -s '[salt]' --proxySalt 'v1.0.0' -
 Change the `-s SALT` to derive a new address. Production deployments use the release version, e.g. `v1.2.1`.
 `proxySalt` is used to derive the same address as a deployment on an existing chain.
 
+### Hyperliquid Block Size Support
+
+For Hyperliquid chains, you can manually switch between BIG and SMALL blocks using the `hyperliquid-block-helper.js` script. This is particularly useful for deploying large contracts like the Gateway contract. For the ITS deployment the module is importer directly into `deploy-its.js` and block changes are handled internally.
+
+To switch block sizes the CHAINS and PRIVATE_KEY should be set in the .env:
+
+```bash
+# Switch to BIG blocks
+node evm/hyperliquid-block-helper.js --blockSize big
+
+# Switch to SMALL blocks  
+node evm/hyperliquid-block-helper.js --blockSize small
```

Comment:
Rename this to `hyperliquid.js` since it can be a generic hyperliquid util
```suggestion
node evm/hyperliquid.js block-size big

node evm/hyperliquid.js block-size small
```

Reply:
updating deployer on ITS token can be added to this file as well